admin/database/postgres/migrations/0007.sql

```diff
@@ -1 +1,8 @@
 ALTER TABLE projects ADD COLUMN subpath TEXT NOT NULL DEFAULT '';
+
+ALTER TABLE user_auth_tokens ADD COLUMN representing_user_id UUID NOT NULL REFERENCES users (id) ON DELETE CASCADE;
+ALTER TABLE user_auth_tokens ADD COLUMN expiration_ts TIMESTAMPTZ;
+
+-- Hard-coded first-party auth clients
+INSERT INTO auth_clients (id, display_name)
+VALUES ('12345678-0000-0000-0000-000000000003', 'Rill Support');
```

Comment:
1. Remove `NOT NULL` constraint
2. `expires_on` instead of `expiration_ts`
3. The migrations should be in a new migration file, old ones won't be run again